setup.py

```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 INSTALL_REQUIRES = [
-    'Wagtail>=2.0',
+    'Wagtail>=2.15',
```

Comment:
**Suggestion** I have a feeling that this version may not be low enough for this particular package. Hopefully an admin could comment here.

Reply:
Is there a technical reason for only supporting Wagtail>2.15 here?

Reply:
Wagtail versions <2.15 are end-of-life. 
https://endoflife.date/wagtail
It's better to bump up the requirements so it would be easier to maintain.

Reply:
I think we should keep INSTALL_REQUIRES set to 2.0, since this package will still work. We don't want to prevent people from using our latest package just because they are on an old Wagtail version.
We can still only support people using 2.15 and above with fixes etc.